project/src/main/scala/org/broadinstitute/clio/sbt/Dependencies.scala

```diff
@@ -169,6 +170,12 @@ object Dependencies {
   val DataaccessModelDependencies: Seq[ModuleID] =
     DataaccessModelMainDependencies ++ DataaccessModelTestDependencies
 
+  val IntegrationTestkitDependencies: Seq[ModuleID] = Seq(
+    "com.dimafeng" %% "testcontainers-scala" % TestContainersScalaVersion,
+    "com.github.pathikrit" %% "better-files" % BetterFilesVersion,
```

Comment:
Don't we use `better-files` all over the place? why do we need to explicitly declare it for integration tests?

Reply:
The new module doesn't depend on any of the other Clio subprojects.